lib/drivers/medtronic/medtronicSimulator.js

```diff
@@ -83,6 +83,17 @@ exports.make = function(config){
     events.push(e);
   }
 
+  function addDurationToTime(time, timezoneOffset, duration) {
```

Comment:
You know, I had it like that and then changed it to use with `.time` as well. You're right in that the gymnastics are only necessary for `.deviceTime`, but I'm so scared of using `Date.parse` now that I only use it where it definitely can't cause any problems (e.g. when subtracting `Date.parse` from `Date.parse` to get a duration).

Is there a reason to not use sundial for `.time`, apart from the extra gymnastics?